apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -1,23 +1,28 @@
 import { useEffect, useMemo, useState } from 'react';
-import { ChannelTypeEnum, IConfigCredentials, ICredentialsDto } from '@novu/shared';
-import { ActionIcon, Group, Loader, Center, Stack } from '@mantine/core';
-import { useClipboard } from '@mantine/hooks';
+import { Group, Center, Box } from '@mantine/core';
 import styled from '@emotion/styled';
 import slugify from 'slugify';
 import { useNavigate, useParams } from 'react-router-dom';
-import { Controller, useForm } from 'react-hook-form';
-import { Button, colors, Input, NameInput, Switch, Text } from '../../design-system';
-import { Check, Close, Copy, DisconnectGradient } from '../../design-system/icons';
+import { Controller, FormProvider, useForm } from 'react-hook-form';
+import { IConfigCredentials, ICredentialsDto, InAppProviderIdEnum } from '@novu/shared';
+
+import { Button, colors, Sidebar, Text } from '../../design-system';
 import { useProviders } from './useProviders';
 import { IIntegratedProvider } from './IntegrationsStorePage';
 import { IntegrationInput } from './components/IntegrationInput';
-import { IntegrationChannel } from './components/IntegrationChannel';
-import { CHANNEL_TYPE_TO_STRING } from '../../utils/channels';
-import { IntegrationEnvironmentPill } from './components/IntegrationEnvironmentPill';
 import { useFetchEnvironments } from '../../hooks/useFetchEnvironments';
-import { ProviderImage } from './components/multi-provider/SelectProviderSidebar';
-import { When } from '../../components/utils/When';
 import { useUpdateIntegration } from '../../api/hooks/useUpdateIntegration';
+import { successMessage } from '../../utils/notifications';
+import { UpdateIntegrationSidebarHeader } from './components/UpdateIntegrationSidebarHeader';
+import { SetupWarning } from './components/SetupWarning';
+import { UpdateIntegrationCommonFields } from './components/UpdateIntegrationCommonFields';
+import { NovuInAppFrameworks } from './components/NovuInAppFrameworks';
+import { FrameworkEnum } from '../quick-start/consts';
+import { When } from '../../components/utils/When';
+import { SetupTimeline } from '../quick-start/components/SetupTimeline';
+import { Faq } from '../quick-start/components/QuickStartWrapper';
+import { NovuInAppFrameworkHeader } from './components/NovuInAppFrameworkHeader';
```

Comment:
Could we have an index export of all the components relative to the integrations pages for an easier management?

Reply:
this will be a lot of changes, I wish we do that from the beginning